<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="2" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-01-30T03:36:42Z" name="Suite" finished-at="2022-01-30T03:37:04Z" duration-ms="22209">
    <groups>
    </groups>
    <test started-at="2022-01-30T03:36:42Z" name="Test" finished-at="2022-01-30T03:37:04Z" duration-ms="22209">
      <class name="TestCases.AddPayeeDatadriven">
        <test-method is-config="true" signature="launchBrowser()[pri:0, instance:TestCases.AddPayeeDatadriven@4d5b6aac]" started-at="2022-01-29T22:36:42Z" name="launchBrowser" finished-at="2022-01-29T22:36:50Z" duration-ms="7631" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method signature="login()[pri:1, instance:TestCases.AddPayeeDatadriven@4d5b6aac]" started-at="2022-01-29T22:36:50Z" name="login" finished-at="2022-01-29T22:36:55Z" duration-ms="5061" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="addPayee(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:TestCases.AddPayeeDatadriven@4d5b6aac]" started-at="2022-01-29T22:36:56Z" name="addPayee" data-provider="PayeeData" finished-at="2022-01-29T22:36:58Z" duration-ms="2554" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Payee1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[brampton20 ]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[abbbb]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[hydro]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addPayee -->
        <test-method signature="addPayee(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:TestCases.AddPayeeDatadriven@4d5b6aac]" started-at="2022-01-29T22:36:59Z" name="addPayee" data-provider="PayeeData" finished-at="2022-01-29T22:37:00Z" duration-ms="1356" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Payee2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[ brampton]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[abbbb]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[hydro]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [The new payee Payee1 was successfully created.] but found [The new payee Payee2 was successfully created.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [The new payee Payee1 was successfully created.] but found [The new payee Payee2 was successfully created.]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at TestCases.AddPayeeDatadriven.addPayee(AddPayeeDatadriven.java:83)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addPayee -->
        <test-method signature="addPayee(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:TestCases.AddPayeeDatadriven@4d5b6aac]" started-at="2022-01-29T22:37:01Z" name="addPayee" data-provider="PayeeData" finished-at="2022-01-29T22:37:02Z" duration-ms="1466" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Payee3]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[brampton20 ]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[abbbb]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[hydro]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [The new payee Payee1 was successfully created.] but found [The new payee Payee3 was successfully created.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [The new payee Payee1 was successfully created.] but found [The new payee Payee3 was successfully created.]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at TestCases.AddPayeeDatadriven.addPayee(AddPayeeDatadriven.java:83)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addPayee -->
        <test-method signature="addPayee(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:TestCases.AddPayeeDatadriven@4d5b6aac]" started-at="2022-01-29T22:37:02Z" name="addPayee" data-provider="PayeeData" finished-at="2022-01-29T22:37:04Z" duration-ms="1372" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Payee4]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[brampton20 ]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[abbbb]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[hydro]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [The new payee Payee1 was successfully created.] but found [The new payee Payee4 was successfully created.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [The new payee Payee1 was successfully created.] but found [The new payee Payee4 was successfully created.]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at TestCases.AddPayeeDatadriven.addPayee(AddPayeeDatadriven.java:83)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addPayee -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:TestCases.AddPayeeDatadriven@4d5b6aac]" started-at="2022-01-29T22:37:04Z" name="closeBrowser" finished-at="2022-01-29T22:37:04Z" duration-ms="291" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- TestCases.AddPayeeDatadriven -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
